### ISSUE TITLE
Key range

### ISSUE THREAD

**Issue by KrashKrash** [JeanLucPons/Kangaroo#123]
After testing it out for a week, i realize this works only if you know the key range. but if you do not know the key range, then it takes forever. so, yah.. nope.

**Comment by Kempeter** [comment]
Yes, but if you can understand the problem, then you can make the code more efficient.

Therefore is open source.

**Comment by Mahuro** [comment]
So.. there are many addresses out there and we can see the PubKey by looking at the transactions. So how do we know what range to use to begin with? I mean, at least a generic starting point range?

**Comment by marssystems** [comment]
> So.. there are many addresses out there and we can see the PubKey by looking at the transactions. So how do we know what range to use to begin with? I mean, at least a generic starting point range?

Start in range 256 - 50% of all private keys are in that range.
